tap-gui/tap-gui-rbac/setup-tap-gui-rbac-eks.md

```diff
@@ -0,0 +1,157 @@
+
+The process for adding access-controlled visibility for a remote EKS cluster is the following:
+
+1. Set up the OIDC provider (pre-requisite)
+2. Configure the EKS cluster with the OIDC provider
+3. Configure the Tanzu Application Platform GUI to view the remote EKS cluster
+4. Upgrade the Tanzu Application GUI package
+
+Once these steps are complete, you can view your runtime resources on a remote EKS cluster in Tanzu Application Platform GUI. For more detail, please refer to [View Runtime Resources on Remote Clusters in Tanzu Application Platform GUI](./view-resouces-rbac.md).
+
+## <a id="set-up-oidc-provider"></a> Set up the OIDC provider
+
+To enable RBAC visibility of remote EKS clusters, you need to set up the OIDC provider. You can see the list of supported OIDC providers in the [Setting up a Tanzu Application Platform GUI authentication provider](./../auth.md) section.
+
+Tanzu Application Platform GUI supports multiple OIDC providers. For the purposes of this guide, we shall use Auth0 as an example.
+
+1. Log in to the Auth0 dashboard
+2. Navigate to Applications
+3. Create an Application
+   - Name: `TAP-GUI` (or your custom app name)
+   - Application type: `Single Page Web Application`
+1. Click on the `Settings` tab
+2. Add under Application URIs > Allowed Callback URLs : `http://tap-gui.INGRESS-DOMAIN/api/auth/auth0/handler/frame`
+   
+    Where:
+
+    - `INGRESS-DOMAIN` is the domain you have chosen for your Tanzu Application Platform GUI in [Installing the Tanzu Application Platform package and profiles](install.md.hbs).
+
+
+1. Click `Save Changes`
+
+After creating an Application with your OIDC provider, you will get the following credentials for setting up RBAC for your remote cluster:
+
+* Domain - to be used as `issuerURL` in the sections below
+* Client ID - to be used as `CLIENT-ID` in the sections below
+* Client Secret - to be used as `CLIENT-SECRET` in the sections below
+
+For more details, please refer to [Auth0 Setup Walkthrough](https://backstage.io/docs/auth/auth0/provider) in the Backstage documentation. To configure other OIDC providers, please refer to [Authentication in Backstage](https://backstage.io/docs/auth/).
+
+## <a id="configure-cluster"></a> Configure the Kubernetes cluster with the OIDC provider
+
+These steps will help you to configure the cluster with the OIDC provider's credentials.
+
+1. Create a file with the contents below and name it `rbac-setup.yaml`. The following example applies to EKS clusters.
+
+```console
+apiVersion: eksctl.10/vialphas
+kind: ClusterConfig
+metadata:
+    name: "CLUSTER-NAME"
+    region: "AWS-REGION"
+identitvProviders:
+  - name: autho
+    type: oidc
+    issuerUrl: "ISSUER-URL"
+    clientId: "CLIENT-ID"
+    usernameClaim: email
+```
+Where:
+   - `CLUSTER-NAME` is the cluster name for your EKS cluster
+   - `AWS-REGION` is the AWS region of the EKS cluster
+   - `ISSUER-URL` is the Issuer URL you obtained while setting up the OIDC provider
```

Comment:
@dbravovmw in the case of Auth0 I think there needs to be a trailing forward slash on this?

Reply:
Auth0 is a little trickier in this point. The actual value should be "https://${AUTH0_DOMAIN}/"

Reply:
`- name: auth0` in the YAML snippet should actually be `- name: https://${AUTH0_DOMAIN}/)`?

Reply:
@dbravovmw , I'll add the following explanation where applicable then:

*Note:* for Auth0, the format of the `issuerURL` is `https://${AUTH0_DOMAIN}/`